src/agentscope/constants.py

```diff
@@ -22,7 +22,7 @@
 
 # for model wrapper
 _DEFAULT_MAX_RETRIES = 3
-_DEFAULT_MESSAGES_KEY = "inputs"
+_DEFAULT_MESSAGES_KEY = "messages"
```

Comment:
Why change the default key?

Reply:
In our recent survey, the most model APIs use the argument name as `messages` rather than `inputs`, so we should consider to use `messages`